.github/workflows/snapshot-mapping-data-copy.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'snapshot-policy-data-copy'
+
+on:
+  schedule:
+    # Cron job every 12 hrs
+    - cron:  '0 */12 * * *'
+  workflow_dispatch:
+
+# Don't cancel in progress since we don't want to have half-baked file change snapshot.
+concurrency: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-snapshot-policy-data-copy'
+
+jobs:
+  snapshot:
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    uses: 'abcxyz/pmap/.github/workflows/snapshot-file-copy.yml@v0.0.5-rc' # Change here to pin the tag or commit.
+    with:
+      workload_identity_provider: '${{ vars.workload_identity_provider }}'
+      service_account: '${{ vars.service_account }}'
+      destination_prefix: '${{ vars.gcs_bucket }}/mapping'
+      path: 'example_org/mapping' #Change here if you change the folder to store the mappings.
```

Comment:
The workflow name says it's for policy but here says "mapping"

Reply:
I changed the workflow name.